app/herdbook.py

```diff
@@ -230,41 +233,102 @@ def herd(h_id):
     return jsonify(data)
 
 
-@APP.route("/api/individual/<i_id>")
+@APP.route("/api/individual/<i_number>")
 @login_required
-def individual(i_id):
+def individual(i_number):
     """
-    Returns information on the individual given by `i_id`.
+    Returns information on the individual given by `i_number`.
     """
     user_id = session.get("user_id", None)
-    ind = da.get_individual(i_id, user_id)
+    ind = da.get_individual(i_number, user_id)
+
     if ind:
-        ind["inbreeding"] = "%.2f" % (get_inbreeding(ind['id']) * 100)
+        ind["inbreeding"] = "%.2f" % (get_ind_inbreeding(ind['number'], ind['genebank_id']) * 100)
```

Comment:
```suggestion
        ind["inbreeding"] = "%.2f" % (get_ind_inbreeding(i_number, ind['genebank_id']) * 100)
```
no need to lookup the number in the dict if the functions already take the individual number